js/src/pro/bitget.js

```diff
@@ -588,6 +588,7 @@ export default class bitget extends bitgetRest {
         }
         let type = undefined;
         [type, params] = this.handleMarketTypeAndParams('watchOrders', market, params);
+        const isStop = this.safeValue(params, 'stop', false);
```

Comment:
@dmunozfer I will update it no worries, but next time please change the correspondent typescript (.ts) file instead, the JS version is automatically compiled by tsc.